components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/openidconnect/OpenIDConnectClaimFilterImpl.java

```diff
@@ -525,6 +527,18 @@ private void handleRolesClaim(Map<String, Object> returnClaims) {
         }
     }
 
+    private void handleGroupsClaim(Map<String, Object> returnClaims) {
```

Comment:
Can we write a unit test covering this improvement?

Reply:
Sure. I will write a unit case